language/move-borrow-graph/src/references.rs

```diff
@@ -44,10 +44,16 @@ pub(crate) struct BorrowEdge<Loc: Copy, Lbl: Clone + Ord> {
     pub(crate) loc: Loc,
 }
 
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub(crate) struct BorrowEdgeSet<Loc: Copy, Lbl: Clone + Ord> {
+    edges: BTreeSet<BorrowEdge<Loc, Lbl>>,
+    overflown: bool,
```

Comment:
I think this needs some explanation

Reply:
lol I was trying not too at the time, but sure I'll add a high-level one